packages/runtime-core/src/hydrationStrategies.ts

```diff
@@ -1,6 +1,27 @@
-import { isString } from '@vue/shared'
+import { getGlobalThis, isString } from '@vue/shared'
 import { DOMNodeTypes, isComment } from './hydration'
 
+const globalThis = getGlobalThis()
+// https://github.com/nuxt/nuxt/blob/main/packages/nuxt/src/app/compat/idle-callback.ts
+// Polyfills for Safari support
+// https://caniuse.com/requestidlecallback
+const requestIdleCallback: typeof globalThis.requestIdleCallback =
+  globalThis.requestIdleCallback ||
+  (cb => {
+    const start = Date.now()
+    const idleDeadline = {
```

Comment:
unused variable

Reply:
Yes, as I asked in your other review, I wonder if in stead of using it plainly like this, we could perhaps rework how it's used in the hydration to make use of the object? Because otherwise this will not behave like idle callback, it would be executed immediately.

Reply:
It's a polyfill so it simply cannot behave like `requestIdleCallback`. The timing object in the polyfill is just there to make the callback arguments consistent, it doesn't change the fact that it's really just a `setTimeout(cb, 1)`.

Reply:
I see, thanks for the explanation!